dkg/frostp2p.go

```diff
@@ -122,6 +126,10 @@ func newFrostP2P(tcpNode host.Host, peers map[peer.ID]cluster.NodeIdx, secret *k
 	p2p.RegisterHandler("frost", tcpNode, round1P2PID,
 		func() proto.Message { return new(pb.FrostRound1P2P) },
 		func(ctx context.Context, pID peer.ID, req proto.Message) (proto.Message, bool, error) {
+			if req == nil {
```

Comment:
same here, the contract of `RegisterHandler` is that it will never pass in nil

Reply:
will revert this too